code/modules/mob/living/life.dm

```diff
@@ -195,7 +195,7 @@
 /mob/living/proc/update_sight()
 	set_sight(0)
 	set_see_in_dark(0)
-	if(stat == DEAD || eyeobj)
+	if(stat == DEAD)
```

Comment:
Can some tell me why this even is a thing? it litervly fires on every tick of life and it seems quiet dumb

Reply:
I figured it out and I hate that this is how it works